setup/install/environment.md

```diff
@@ -87,6 +87,7 @@ Ensure that the following are installed on your system:
 * git
 * curl
 * yarn ([guide](https://yarnpkg.com/lang/en/docs/install/))
+* node (version >=8.9.0, [can be installed via nvm](https://github.com/creationix/nvm#install-script))
```

Comment:
You should install node & npm first, which you can then install yarn using `npm` (I had to do this yesterday)

Reply:
Ah nice, sounds good. Can you submit a PR with the steps you took yesterday?